src/PKSim.CLI.Core/RunOptions/QualificationRunOptions.cs

```diff
@@ -16,5 +16,10 @@ public class QualificationRunOptions
       ///    Should simulation be performed as part of the run?
       /// </summary>
       public bool Run { get; set; }
+
+      /// <summary>
+      /// Specifies if project files (snapshot and project should be exported)
+      /// </summary>
+      public bool ExportProjectFiles { get; set; }
```

Comment:
Should we export the project files or not. Default will be not because I don't think we need to export those for the qualification

Reply:
ok, let's keep it like this. 
If this option can be passed to the QualificationRunner later I'm perfectly fine with that :)